h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -183,6 +192,12 @@ public GLMModel clone2(){
     public Solver _solver = Solver.AUTO;
     public double _tweedie_variance_power;
     public double _tweedie_link_power;
+    public double _theta; // 1/k and is used by negative binomial distribution only
+    public double _invTheta;
+    public double _trainDataTheta;  // stored the theta from training data
```

Comment:
will be removed according to @wendycwong

Reply:
Done